source/nodejs/adaptivecards/src/carousel.ts

```diff
@@ -241,6 +247,15 @@ export class Carousel extends Container {
         const containerForAdorners: HTMLElement = document.createElement("div");
         containerForAdorners.className = this.hostConfig.makeCssClassName("ac-carousel-container");
 
+        if (this.header) {
+            this.header.setParent(this);
+            let renderedHeader = this.header?.render();
+            if (renderedHeader) {
+                Utils.appendChild(cardLevelContainer, renderedHeader);
+                renderedHeader.className = this.hostConfig.makeCssClassName("ac-carousel-header");
```

Comment:
genuinely asking, because I haven't really thought it through -- should we be _replacing_ the `ac-richTextBlock` `className` with `ac-carousel-header`?